src/sentry/api/serializers/models/event.py

```diff
@@ -134,6 +139,10 @@ def serialize(self, obj, attrs, user):
             'dateCreated': obj.datetime,
             'dateReceived': received,
             'errors': errors,
+            'fingerprints': map(
```

Comment:
In theory, this won't be python3 compat since this will return a generator, and will probably break serializer.